pipeline/event.go

```diff
@@ -260,7 +263,8 @@ func (p *eventPool) get() *Event {
 	event := p.events[x]
 	p.events[x] = nil
 	p.free2[x].Store(false)
-
+	metric.GetGauge(p.subsystemName(), workEventsGauge).Inc()
```

Comment:
Can we call metric.GetGauge only once and inc here result of  metric.GetGauge(p.subsystemName(), workEventsGauge) ?

Reply:
We make a choice, either add two counters, and conveniently write metrics to the logs, or leave only the prometheus metric and get the value in the logs through error handling.